Updated pinned pytorch

guangy10 opened the pull request:
Updated pinned torch to a commit from `viable/strict`.

Note that we are now pinning pytorch commit in ExecuTorch `main`. Until we have the auto pin update working, for manual pin update, please use commit either in `main` or `viable/strict`:
  - commits from main: https://hud.pytorch.org/
  - commit from viable/strict:  https://hud.pytorch.org/hud/pytorch/pytorch/viable%2Fstrict/1?per_page=50


Differential Revision: D51992477

huydhn reviewed (approved):
LGTM! You'll experience first hand the increase CI time to build PyTorch as part of the process to build Docker image here https://github.com/pytorch/executorch/actions/runs/7145784936.  It's an one time cost for each update.

guangy10 committed:
Updated pinned pytorch (#1384)

Summary:

We are now pinning pytorch commit in ExecuTorch main. Until we have the auto pin update working, for manual pin update, please use commit either in main or viable/strict:
 - commits from `main`: https://hud.pytorch.org/
 - commit from `viable/strict`:  https://hud.pytorch.org/hud/pytorch/pytorch/viable%2Fstrict/1?per_page=50

Differential Revision: D51992477
